Python/ceval.c

```diff
@@ -738,18 +738,17 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
 
 /* Sets the above local variables from the frame */
 #define SET_LOCALS_FROM_FRAME() \
```

Comment:
There's only one call site left for this macro -- can we just expand it there? I think it would make the code clearer, and this doesn't really encapsulate a very useful abstraction any more.